modules/db_instance/main.tf

```diff
@@ -70,6 +70,11 @@ resource "aws_db_instance" "this" {
   apply_immediately           = var.apply_immediately
   maintenance_window          = var.maintenance_window
 
+  # https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/blue-green-deployments.html
+  blue_green_update {
```

Comment:
lets use the expanded form so that any additional feature additions won't be a breaking change:

```hcl
  dynamic "blue_green_update" {
    for_each = length(var.blue_green_update) > 0 ? [var.blue_green_update] : [] 
    
    content {
      enabled = try(blue_green_update.value.enabled, null)
    }
  }
...